miden-lib/asm/kernels/transaction/lib/epilogue.masm

```diff
@@ -161,6 +163,56 @@ proc.update_account_code
     # => []
 end
 
+
+# ACCOUNT STORAGE UPDATE
+# =================================================================================================
+
+#! Updates the account storage commitment by hashing the storage slots and setting the storage
+#! commitment to the newly computed storage hash.
+#!
+#! Stack: []
+#! Output: []
+proc.update_account_storage
```

Comment:
nit: I would rename this into `update_account_storage_commitment` to make it clear that we are not actually updating the storage itself.

Reply:
Updated + renamed all `storage root` to `storage commitment`